test/functional/pruning.py

```diff
@@ -307,15 +313,15 @@ def has_block(index):
 
         # stop node, start back up with auto-prune at 550MB, make sure still runs
         self.stop_node(node_number)
-        self.nodes[node_number] = self.start_node(node_number, self.options.tmpdir, ["-prune=550"], timewait=900)
+        self.start_node(node_number, extra_args=["-prune=550"])
```

Comment:
You are removing `timewait=900` from all calls but the first. I assume this does not cause issues?

Reply:
`rpc_timeout` is a property of `TestNode` set when instantiating the python object, not when starting up the bitcoind node.